inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_split.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "dsr_tests_common.hpp"
+
+namespace {
+
+using namespace LayerTestsUtils::vpu;
+
+struct SplitTestCase {
+    DataShapeWithUpperBound dataShapes;
+    int64_t axis, numSplits;
+};
+
+const auto combinations = testing::Combine(
+        testing::Values(
+                ngraph::element::f16),
+        testing::Values(
+                ngraph::element::i32),
+        testing::Values(
+                SplitTestCase{{{6, 12, 10}, {6, 12, 15}}, 1, 3},
+                SplitTestCase{{{6, 12}, {10, 12}}, 1, 4},
+                SplitTestCase{{{6, 12, 10, 24}, {6, 12, 10, 50}}, 0, 6},
+                SplitTestCase{{{6, 12, 10, 24}, {6, 12, 10, 50}}, -3, 2}),
```

Comment:
Are these cases from FasterRCNN? Only dynamism at the last dimension is supported?

Reply:
It's not, these cases are actually based on tests for VariadicSplit, but I've forgotten to add the tests from the network where it was initially appeared (it's mobilenet-ssdlite), it has the last but one dynamic dimension, I've also added case where the first dimension is dynamic.